packages/desktop-client/src/components/budget/util.ts

```diff
@@ -67,14 +67,20 @@ export function makeAmountStyle(
   }
 }
 
-export function makeAmountFullStyle(value: number) {
+export function makeAmountFullStyle(
+  value: number,
+  colors?: {
+    positiveColor?: string;
+    negativeColor?: string;
+    zeroColor?: string;
+  },
+) {
+  const _positiveColor = colors.positiveColor || theme.noticeText;
+  const _negativeColor = colors.negativeColor || theme.errorText;
+  const _zeroColor = colors.zeroColor || theme.tableTextSubdued;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏tbh I'm not a big fan of this naming convention. I don't see it adding any value. Especially in this context.